### DISCUSSION TITLE
Just installed and can't login!

### DISCUSSION THREAD

**Discussion by geniushostuk** [LiveHelperChat/livehelperchat#2160] [General]
Great to be here. I eventually found my way here from your old support forum.

I've just installed version 4.51, tried to login and nothing happens. There's no message of invalid username/password but it doesn't login and just sits there.

Hopefully this is a quick fix?

**Comment by remdex** [comment]
I ques your php sessions does not work...

**Reply by geniushostuk** [reply]
Thanks for the prompt response.

That's entirely possible as I had to ask the web host to add php version 8.2

Is there a log file that will give further info?
